util/normalize.lua

```diff
@@ -6,11 +6,15 @@ and add individual player names to recipient list
 --]]
 function mail.normalize_players_and_add_recipients(field, recipients, undeliverable)
     local order = mail.parse_player_list(field)
-    for _, player_name in ipairs(order) do
-        if not minetest.player_exists(player_name) then
-            undeliverable[player_name] = true
-        else
-            recipients[player_name] = true
+    for _, recipient_name in ipairs(order) do
+        if recipient_name:split("@")[1] == "" then -- in case of maillist
+            recipients[recipient_name] = true
```

Comment:
Should this also check if mailing list exists (has at least 1 subscriber) and add to undeliverable if it does not? I mean similar how player check is done but for lists.

I've no idea just yet how mailing lists work but doing all recipient validation here feels like correct thing to do if possible.

Reply:
Yes, I think this line was deleted, thx.